packages/dcos-integration-test/extra/test_legacy_user_management.py

```diff
@@ -175,20 +177,20 @@ def test_user_delete(dcos_api_session):
     assert 'user6@domain.foo' not in users
 
 
-def test_user_put_requires_authentication(noauth_api_session):
+def test_user_put_requires_authentication(noauth_api_session: Any) -> None:
     r = noauth_api_session.put('/acs/api/v1/users/user7@domain.foo', json={})
     assert r.status_code == 401, r.text
 
 
-def test_dynamic_ui_config(dcos_api_session):
+def test_dynamic_ui_config(dcos_api_session: Any) -> None:
     r = dcos_api_session.get('/dcos-metadata/ui-config.json')
     data = r.json()
     assert not data['clusterConfiguration']['firstUser']
     assert 'id' in data['clusterConfiguration']
     assert 'uiConfiguration' in data
 
 
-def test_dcos_add_user(dcos_api_session, new_dcos_cli):
+def test_dcos_add_user(dcos_api_session: Any, new_dcos_cli: Any) -> None:
```

Comment:
The CLI in these arguments has type `dcos_test_utils.dcos_cli.DcosCli`

Reply:
will be replaced in last commit